src/main/scanLogic/scanRunners/binaryRunner.ts

```diff
@@ -92,6 +123,29 @@ export abstract class BinaryRunner {
         await Promise.all(tasksInfo.tasks);
     }
 
+    /**
+     * Create the needed environment variables for the runner to run
+     * @returns list of environment variables to use while executing the runner or unidentified if credential not set
+     */
+    private createEnvForRun(): NodeJS.ProcessEnv | undefined {
+        if (this._connectionManager.areXrayCredentialsSet()) {
+            let binaryVars: NodeJS.ProcessEnv = {
+                JF_PLATFORM_URL: this._connectionManager.url
+            };
+            if (this._connectionManager.accessToken) {
+                binaryVars.JF_TOKEN = this._connectionManager.accessToken;
+            } else {
+                binaryVars.JF_USER = this._connectionManager.username;
+                binaryVars.JF_PASS = this._connectionManager.password;
+            }
+            return {
```

Comment:
We should also populate `HTTP_PROXY` & `HTTPS_PROXY` environment variables.

See this for reference:
https://github.com/jfrog/jfrog-idea-plugin/pull/264/files#diff-1d8599a3661f898652e8899cfbd3361e01354637b5ff73ea5fa41f81464648a8R68-R77

Reply:
done